apps/sync/js/crypto/fxsyncwebcrypto.js

```diff
@@ -0,0 +1,290 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+/* global
+  crypto,
+  KeyDerivation,
+  StringConversion
+*/
+
+/* exported
+  FxSyncWebCrypto
+*/
+
+// WebCrypto-based client for Firefox Sync.
+
+var FxSyncWebCrypto = (() => {
+
+  const HKDF_INFO_STR = 'identity.mozilla.com/picl/v1/oldsync';
+
+  var FxSyncWebCrypto = function() {
```

Comment:
Call me crazy, but if you're using `=>` (and are thus using modern JS), why are you using `var`?